src/index.js

```diff
@@ -0,0 +1,195 @@
+  } else if (temperature >= 70 && temperature <= 79) {
+    return "orange";
+  } else if (temperature >= 60 && temperature <= 69) {
+    return "yellow";
+  } else if (temperature >= 50 && temperature <= 59) {
+    return "green";
+  } else if (temperature <= 49) {
+    return "teal";
+  }
+};
+
+const tempColorUpdate = () => {
+  state.temperatureValue.className = getTemperatureColor(
+    state.currentTemperature
+  );
+};
+
+const increaseTemperature = () => {
+  state.currentTemperature += 1;
+  state.temperatureValue.textContent = state.currentTemperature;
+  allGraphicsUpdate();
+};
+
+const decreaseTemperature = () => {
+  state.currentTemperature -= 1;
+  state.temperatureValue.textContent = state.currentTemperature;
+  allGraphicsUpdate();
+};
+
+const locUrl = "http://localhost:5000/location";
+const weatherUrl = "http://localhost:5000/weather";
+
+const getTempFromLocation = () => {
+  const q = state.cityTextField.value;
+  axios
+    .get(locUrl, { params: { q } })
+    .then((response) => {
+      const lat = response.data[0].lat;
+      const lon = response.data[0].lon;
+      axios
+        .get(weatherUrl, { params: { lat, lon } })
+        .then((weatherResponse) => {
+          const tempCelsius = weatherResponse.data.main.temp - 273.15;
+          const newTemp = Math.floor(tempCelsius * (9 / 5) + 32);
+          state.currentTemperature = newTemp;
+          state.temperatureValue.textContent = state.currentTemperature;
+          const mainWeather = weatherResponse.data.weather[0].main;
+          console.log(mainWeather);
+          if (mainWeather in weatherOptions) {
+            state.skyOptions.value = weatherOptions[mainWeather];
+            console.log(weatherOptions[mainWeather]);
+          } else {
+            state.skyOptions.value = 1;
+          }
+
+          allGraphicsUpdate();
+        })
+        .catch((error) => {
+          console.log("Weather call failed", error);
+        });
+    })
+    .catch((error) => {
+      console.log("Location call failed", error);
+    });
+};
+
+const allGraphicsUpdate = () => {
+  tempColorUpdate();
+  landscapeUpdate();
+  skyOptionsUpdate();
+};
+
+const loadData = () => {
+  state.cityTextField = document.querySelector("#cityNameInput");
+  state.headerCity = document.querySelector("#headerCityName");
+  state.cityResetButton = document.querySelector("#cityNameReset");
+  state.landscapeField = document.querySelector("#landscape");
+  state.skyField = document.querySelector("#sky");
+  state.skyOptions = document.querySelector("#skySelect");
+  state.currTempButton = document.querySelector("#currentTempButton");
+  state.temperatureValue = document.querySelector("#tempValue");
+  state.temperatureIncreaseButton = document.querySelector(
+    "#increaseTempControl"
+  );
+  state.temperatureDecreaseButton = document.querySelector(
+    "#decreaseTempControl"
+  );
+};
+
+const registerEventHandlers = (event) => {
+  loadData();
+  allGraphicsUpdate();
```

Comment:
Consider refactoring this code to _only_ have the event registration in this function. We can make another function (maybe `initializePage`) that looks up the controls, registers the events, and updates the page display. Then `initializePage` is what we would register with the `DOMContentLoaded` event.